src/js/stores/AppFormStore.js

```diff
@@ -173,7 +173,8 @@ const responseAttributePathToFieldIdMap = {
   "/labels": "labels",
   "/ports": "ports",
   "/uris": "uris",
-  "/user": "user"
+  "/user": "user",
+  "value": "general"
```

Comment:
@pierlo-upitup I did address the general issue from [#3241](https://github.com/mesosphere/marathon/issues/3241). You can have a look at the list of changes I applied in PR [#3255](https://github.com/mesosphere/marathon/pull/3255). I am not sure if you would like to reverse the mapping from `value` to `general`, but if you do so, I would like to remind you of that here. If you have any feedback on recent changes, please do not hesitate to approach me.